Modules/_posixsubprocess.c

```diff
@@ -30,6 +30,12 @@
 # define SYS_getdents64  __NR_getdents64
 #endif
 
+#if defined(__linux__) && defined(HAVE_SIGNAL_H) && \
+    defined(HAVE_PTHREAD_SIGMASK) && !defined(HAVE_BROKEN_PTHREAD_SIGMASK)
+# include <signal.h>
+# define VFORK_USABLE 1
```

Comment:
We may need to add a configure test for `vfork` if possibility of Linux without `vfork` is considered worthy.